clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -82,13 +82,22 @@
   </xs:complexType>
 
   <xs:complexType name="connection-spec">
-    <xs:attribute name="url" type="tc:connectionUrl" use="required">
+    <xs:attribute name="url" type="tc:connectionUrl">
       <xs:annotation>
         <xs:documentation xml:lang="en">
           Identifies a single cluster member by URL.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="servers" type="tc:servers">
```

Comment:
Why not make this syntax mirror the syntax we use in tc-config (at least structurally).

Reply:
Will check.

Reply:
Done. (Ditto for other xsd-related comments as well).

Now the servers can be specified as:
```
<servers-connection>
  <servers>
    <server host="some.host" port="1234">
    <server host="other.host" port="1234">
    ...
  </servers>
  <cluster-tier-manager name="cM"/>
</servers-connection>
```

Note that the above is mutually exclusive (within the schema) with `<connection url="terracotta://some.host/cM" />`.

I had to create the new tag `<servers-connection>` because I couldn't find a way to specify mutually exclusive (at the XSD level) attributes and sub elements. I was look for
`<connection url="terracotta://some.host/cM" />` , OR
`<connection>
  <servers>
    <server host="some.host" port="1234">
    <server host="other.host" port="1234">
    ...
  </servers>
  <cluster-tier-manager name="cM"/>
</connection>`

If there is indeed a way to do the above in XSD, please let me know. If not, the current approach should be fine. I'm open to naming suggestions to replace `<servers-connection>`.

Reply:
I think the mutually exclusive attribute and element are indeed impossible. If I where you I'd take out a nesting level on the server list though leaving something like:

```xml
<connection url="terracotta://some.host/cM" />
```
or
```xml
<cluster cluster-tier-manager="cM">
  <server host="some.host" port="1234">
  <server host="other.host" port="1234">
</cluster>
```

(Naming obviously flexible)

Reply:
I made it to:
```
<cluster-connection cluster-tier-manager="cM">
  <server host="some.host" port="1234">
  <server host="other.host" port="1234">
</cluster-connection>
```
because this element is a sub-element of `<cluster>`.